commit 027a7707bea5ba455212715278e3ae486f97eacb
Author: Doug Goldstein <cardoe@cardoe.com>
Date:   Thu Jun 27 20:43:00 2013 -0500

    Allow RO connections to interface udev backend
    
    The udev based interface backend did not allow querying data over a
    read-only connection which is different than how the netcf backend
    operates. This brings the behavior inline with the default, netcf
    backend.

diff --git a/src/interface/interface_backend_udev.c b/src/interface/interface_backend_udev.c
index 68e1e2f..95034bf 100644
--- a/src/interface/interface_backend_udev.c
+++ b/src/interface/interface_backend_udev.c
@@ -139,7 +139,7 @@ udevInterfaceOpen(virConnectPtr conn,
 {
     struct udev_iface_driver *driverState = NULL;
 
-    virCheckFlags(0, VIR_DRV_OPEN_ERROR);
+    virCheckFlags(VIR_CONNECT_RO, VIR_DRV_OPEN_ERROR);
 
     if (VIR_ALLOC(driverState) < 0) {
         virReportOOMError();
